### PR TITLE
Add inspirational call-to-action

### PR DESCRIPTION
**Pull request by jkwok91**
with the appropriate styling

**This needs to be tested; I haven't gotten website running locally yet.**
but I might forget about this if I don't make the PR

### CODE REVIEW

**Comment by MaxWofford** [comment]
@jkwok91 Great call to action!

File: app/views/pages/home.html.haml [hunk 1]
```diff
@@ -136,6 +136,9 @@
 .email-form.small-section
   .row
     %h1 Stay in the Loop
+    %p.call-to-action
+      Join hundreds of amazing people across the world who are banding together
+to build the communities they always wished existed.
```
Comment (MaxWofford) [inline app/views/pages/home.html.haml]: @jkwok91 Haml expects this line to  be indented like the one before it.

**Comment by jkwok91** [comment]
actually it's @jonleung's call to action :smile:

**Comment by MaxWofford** [comment]
screenshot from 2016-02-15 19 26 38

lgtm

**Pull request merged**